inference-engine/src/plugin_api/threading/ie_istreams_executor.hpp

```diff
@@ -68,6 +68,14 @@ class INFERENCE_ENGINE_API_CLASS(IStreamsExecutor) : public ITaskExecutor {
         */
         Parameter GetConfig(const std::string& key);
 
+        /**
+        * @brief Create appropriate maltythreaded configuration
```

Comment:
typo in `maltythreaded`

Can we re-use the same for CPU plugin?

Reply:
Yes it can be reused inside CPU plugin. But I'm going to add it into CPU plugin only with other more valuable changes  when performance validation will be needed.

Reply:
Ok, fix typo in comment and I will merge